build.sh

```diff
@@ -116,6 +120,16 @@ while [[ $# -gt 0 ]]; do
         STATIC_LIBRARY="-DSTATIC_LIBRARY=1"
         ;;
 
+    --without=*)
+        FEATURE=$1
+        if [[ "$WITHOUT_FEATURES" == "" ]]; then
+            WITHOUT_FEATURES="-DWITHOUT_FEATURES="
+        else
+            WITHOUT_FEATURES="$WITHOUT_FEATURES;"
+        fi
+        WITHOUT_FEATURES="${WITHOUT_FEATURES}-DCOMPILE_DISABLE_${FEATURE:10}=1"
```

Comment:
Agree it is not straight forward. This is the same approach used through out the script, need some effort to understand.